### PR TITLE
Add a new listig view for support services

### PR DESCRIPTION
**Pull request by fujimotos**
The support information was a bit old. Let's update it.

Signed-off-by: Fujimoto Seiji <fujimoto@clear-code.com>

### CODE REVIEW

**Comment by fujimotos** [comment]
This is what the new support page looks like:

Screenshot_2021-04-06_15-45-36

File: ja/support/index.md [hunk 1]
```diff
@@ -4,126 +4,16 @@ title: サポート
 
 # サポート
 
-PGroongaは[コミュニティー](../community/)によるサポートの他にも商用サポートサービスがあります。
+以下の企業は、PGroongaのサポートサービスを提供しています（五十音順）
 
-PGroongaを使った検索システムの検討から運用まで[クリアコード](https://www.clear-code.com/)がサポートします。クリアコードにはPGroongaの開発者が在籍しています。
+### 株式会社クリアコード
```
Comment (kou) [inline ja/support/index.md:9]:
```suggestion
## 株式会社クリアコード
```

Comment (fujimotos) [reply ja/support/index.md]: Fixed.

File: ja/support/index.md [hunk 2]
```diff
@@ -4,126 +4,19 @@ title: サポート
...
-      <td>
-        <ul>
-          <li>インシデントサポート 50万円/年～</li>
-          <li>サービスレベル、サポート内容に応じてお見積もりいたします。</li>
-        </ul>
-      </td>
-    </tr>
-  </tbody>
-</table>
+クリアコードは2008年からGroongaの開発に参加しています。PGroonga、[Groonga](https://groonga.org/ja/)、<a href='https://mroonga.org/ja/'>Mroonga</a>を採用したシステム開発、保守の支援を提供しています。
```
Comment (kou) [inline ja/support/index.md:16]:
```suggestion
クリアコードは2008年からGroongaの開発に参加しています。PGroonga、[Groonga](https://groonga.org/ja/)、[Mroonga](https://mroonga.org/ja/)を採用したシステム開発、保守の支援を提供しています。
```

Comment (fujimotos) [reply ja/support/index.md]: Fixed.

**Comment by kou** [comment]
Thanks.
I've merged.

**Pull request merged**